### ISSUE TITLE
pypi is not updated with the latest version?

### ISSUE THREAD

**Issue by kalsky** [jmcgeheeiv/pyfakefs#117]
when checking the pypi index (https://pypi.python.org/pypi/pyfakefs), it shows that the latest version is 2.8
but when trying to install this package, it fails if you specify 2.8 you get an error that it doesn't exist:

```
pip install pyfakefs==2.8
Collecting pyfakefs==2.8
  Could not find a version that satisfies the requirement pyfakefs==2.8 (from versions: 1.1, 1.2, 2.0, 2.1, 2.2, 2.3, 2.4, 2.7)
No matching distribution found for pyfakefs==2.8
```

when not specifying a versions, it gets 2.7

**Comment by jmcgeheeiv** [comment]
Ah, I think that 2.8 vs. 2.7 PyPi problem was my mistake.  When I released 2.9 yesterday, I noticed that I may have skipped a step in the past.

@kalsky, please give version 2.9 a go.  Subject it to your usual torture test and close this issue if you are satisfied.

**Comment by jmcgeheeiv** [comment]
As we have heard no complaints about version 2.9 on PyPi, closing this issue.
